config/initializers/authed_domains.rb

```diff
@@ -0,0 +1,5 @@
+if Rails.application.secrets.basic_auth_credentials
+  LinkCheckerApi.authed_domains = {
```

Comment:
Perhaps this should just set the key in the hash rather than replacing the entire hash.